src/main/java/net/nikr/eve/jeveasset/io/local/SettingsReader.java

```diff
@@ -1548,62 +1667,73 @@ private CompareType convertMode(final String compareMixed) {
 		return CompareType.CONTAINS;
 	}
 
-	private void parseExportSettings(final Element element, final Settings settings) throws XmlException {
+	/***
+	 * Old method to process export settings for 6.8.0 and older
+	 */
+	@Deprecated
+	private void parseExportSettingsLegacy(final Element element, final Settings settings) throws XmlException {
```

Comment:
This really is to only load the old export setting format one time and it is basically never used again. I'm not sure that this functionality is needed. Plus this looks like you would need to save the setting with old first before this method would even fire unless I'm reading something wrong. This would require us to constantly save the old setting indefinitely and wouldn't help us in the first load since nothing would have old in it.

Reply:
Okay, It was just an idea. Lets keep it as is 👍